oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -30,6 +28,7 @@ struct OciFilesystemBundleConfig {
 }
 
 #[derive(serde::Deserialize)]
+#[allow(unused)]
```

Comment:
Why are we allowing unused properties? 

In this case it seems to be since we no longer read the cwd property. It's worth noting that said property appears to be required in the OCI filesystem bundle spec: https://github.com/opencontainers/runtime-spec/blob/v1.0.2/config.md#process

Do we still support setting the cwd?